index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <link rel="stylesheet" type="text/css" href="styles.css" media="screen"/>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/9650a62e47.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="css/style.css">
+  <title>Product Landing Page - iMouse</title>
+</head>
+
+<body>
+  <header id="header">
+    <nav id="nav-bar">
+      <img id="header-img" src="https://images-na.ssl-images-amazon.com/images/I/51uTvI7ew-L._AC_SY450_.jpg" alt="logo" class="logo">
+      <ul>
+        <a class="nav-link" href="#features">Features</a>
+        <a class="nav-link" href="#how-it-work">How It Works</a>
+        <a class="nav-link" href="#pricing">Pricing</a>
+      </ul>
+    </nav>
+  </header>
+

```

Comment:
Kindly remove the extra blank lines and indentations for a cleaner code file.
Please keep in mind that we're only adding blank lines to separate two logical pieces of code or two large sections apart.

Reply:
- I like this, good job!